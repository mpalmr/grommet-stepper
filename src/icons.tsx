import React from 'react';

export const DeincrementIcon: React.FC = () => (
	<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
		<path
			d="M20 8l-8 8-8-8"
			stroke="#fff"
			strokeWidth="2"
			strokeLinejoin="round"
		/>
	</svg>
);

export const IncrementIcon: React.FC = () => (
	<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
		<path
			d="M12 0a1.5 1.5 0 00-1.5 1.5v9h-9a1.5 1.5 0 000 3h9v9a1.5 1.5 0 003 0v-9h9a1.5 1.5 0 000-3h-9v-9A1.5 1.5 0 0012 0z"
			fill="#fff"
		/>
	</svg>
);
