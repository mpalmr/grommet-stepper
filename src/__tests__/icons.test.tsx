import React from 'react';
import { render } from '@testing-library/react';
import { Grommet } from 'grommet';
import { DeincrementIcon, IncrementIcon } from '../icons';

test('<DeincrementIcon /> renders', () => {
	const { container } = render((
		<Grommet>
			<DeincrementIcon />
		</Grommet>
	));
	expect(container).toMatchSnapshot();
});

test('<IncrementIcon /> renders', () => {
	const { container } = render((
		<Grommet>
			<IncrementIcon />
		</Grommet>
	));
	expect(container).toMatchSnapshot();
});
