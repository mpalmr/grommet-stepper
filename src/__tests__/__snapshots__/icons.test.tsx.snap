// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DeincrementIcon /> renders 1`] = `
<div>
  <div
    class="StyledGrommet-sc-19lkkz7-0 djOnwZ"
  >
    <svg
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M20 8l-8 8-8-8"
        stroke="#fff"
        stroke-linejoin="round"
        stroke-width="2"
      />
    </svg>
  </div>
</div>
`;

exports[`<IncrementIcon /> renders 1`] = `
<div>
  <div
    class="StyledGrommet-sc-19lkkz7-0 djOnwZ"
  >
    <svg
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M12 0a1.5 1.5 0 00-1.5 1.5v9h-9a1.5 1.5 0 000 3h9v9a1.5 1.5 0 003 0v-9h9a1.5 1.5 0 000-3h-9v-9A1.5 1.5 0 0012 0z"
        fill="#fff"
      />
    </svg>
  </div>
</div>
`;
